test/easyconfigs/easyconfigs.py

```diff
@@ -578,7 +578,8 @@ def check_python_packages(self, changed_ecs):
 
             # if Python is a dependency, that should be reflected in the versionsuffix
             # Tkinter is an exception, since its version always matches the Python version anyway
-            if any(dep['name'] == 'Python' for dep in ec['dependencies']) and ec.name != 'Tkinter':
+            if any(dep['name'] == 'Python' for dep in ec['dependencies']) and ec.name != 'Tkinter' and \
+                ec.name != 'GObject-Introspection':
```

Comment:
@Flamefire Just a reminder: do open a PR to roll this back after #9528 is merged...

Reply:
Please ping me again when this (finally) will be merged. Hooray for `sed`, this sanity checks  on the old ECs get annoying ;)

Reply:
Thanks a lot for your efforts on this one! :+1:

Reply:
Fixed in #9577, thanks @Flamefire!